common/service.go

```diff
@@ -64,7 +66,7 @@ Typical usage:
 	}
 */
 type BaseService struct {
-	log     log15.Logger
+	Logger  log.Logger
```

Comment:
Yes. We could have a method `GetLogger`, but field is faster and looks cleaner. See https://github.com/tendermint/abci/pull/78